src/ConnectionFactory.php

```diff
@@ -75,4 +83,31 @@ protected function getDefaultConfig($configKey)
             'doctrine_commented_types' => [],
         ];
     }
+
+    /**
+     * Registers all declared typed, if not already done.
+     *
+     * @param ContainerInterface $container
+     */
+    private function registerTypes(ContainerInterface $container)
+    {
+        if (self::$areTypesRegistered) {
+            return;
+        }
+
+        $applicationConfig = $container->has('config') ? $container->get('config') : [];
```

Comment:
Just let it crash if `'config'` is not set - this is a very likely failure scenario that, if made silent, can lead to a lot of hair pulling